src/types/commands.ts

```diff
@@ -518,10 +524,8 @@ export interface UpdateChartCommand extends SheetDependentCommand {
 // Image
 //------------------------------------------------------------------------------
 
-export interface CreateImageOverCommand extends SheetDependentCommand {
+export interface CreateImageOverCommand extends SheetDependentCommand, SubFigureCommand {
```

Comment:
This will require an upgrade script in odoo. See here for an exemple:
https://github.com/odoo/upgrade/blob/master/migrations/spreadsheet_edition/saas~17.5.1.0/pre-migrate.py